modules/virt-creating-localnet-nad-cli.adoc

```diff
@@ -61,15 +61,17 @@ spec:
             "type": "ovn-k8s-cni-overlay", <3>
             "topology": "localnet", <4>
             "mtu": 1500, <5>
-            "netAttachDefName": "default/localnet-network" <6>
+            "vlanID": 200, <6>
+            "netAttachDefName": "default/localnet-network" <7>
     }
 ----
 <1> The CNI specification version. The required value is `0.3.1`.
 <2> The name of the network. This attribute must match the value of the `spec.desiredState.ovn.bridge-mappings.localnet` field of the `NodeNetworkConfigurationPolicy` object that defines the OVS bridge mapping.
 <3> The name of the CNI plug-in to be configured. The required value is `ovn-k8s-cni-overlay`.
 <4> The topological configuration for the network. The required value is `localnet`.
 <5> Optional: The maximum transmission unit (MTU) value. If you do not set a value, the Cluster Network Operator (CNO) sets a default MTU value by calculating the difference among the underlay MTU of the primary network interface, the overlay MTU of the pod network, and byte capacity of any enabled features, such as IPsec.
-<6> The value of the `namespace` and `name` fields in the `metadata` stanza of the `NetworkAttachmentDefinition` object.
+<6> Optionally, for more fine-grained network management, you can configure a virtual LAN (VLAN) ID for the NAD. VMs that use this NAD will have an interface whose traffic will be able to communicate only with other devices that use the same VLAN ID (`200` in this example).
```

Comment:
> VMs that use this NAD will have an interface whose traffic will be able to communicate only with other devices that use the same VLAN ID (`200` in this example).

Avoid using future tense. This sentence is also quite long, and with the current CQA assessments checking for sentence length, it would be flagged.

Maybe something along the lines of:

VMs with this NAD can only communicate with other devices using the same VLAN ID (`200` in this example).

Reply:
This exact wording might be somewhat misleading, I think, but you're right that the meaning can be expressed accurately enough without the future tense.